src/components/VerticalActionsList.js

```diff
@@ -96,6 +105,21 @@ function VerticalActionsList({
       {viewsOpen.priority && (
         <Priority onClose={() => setViews({ ...defaultViews })} />
       )}
+      {viewsOpen.archive && (
+        <AlertPopupTemplate
+          onClose={() => setViews({ ...defaultViews })}
+          className='archive_popup'
+          heading='Archiving'
+          // TODO : make the archiving message dynamic, showing the title(s) of the card(s) selected for archiving //
+          content={archiveMessagePartOne, archiveMessagePartTwo}
```

Comment:
Did you verify that this `archiveMessagePartOne, archiveMessagePartTwo` actually works to display the intended message? By the looks of it to me the syntax wouldn't work.

Reply:
fixed